src/swagger/trip.swagger.js

```diff
@@ -357,3 +357,71 @@
  *         description: Trip ID doesn't exists
  *
  */
+
+/**
+ * @swagger
+ * definitions:
+ *   Edit Pending Trip Request:
+ *     type: object
+ *     properties:
+ *       originId:
+ *         type: integer
+ *       destinationId:
+ *         type: integer
+ *       departureDate:
+ *         type: string
+ *       returnDate:
+ *         type: string
+ *       travelReasons:
+ *         type: string
+ *       accommodationId:
+ *         type: integer
+ */
+
+/**
+ * @swagger
+ * /api/trips/requests/{tripId}/edit-trip-request:
```

Comment:
when you change the route remember to change it here too

Reply:
Ok changed it